utils/unittest/unittest.go

```diff
@@ -320,7 +321,10 @@ func badgerDB(t testing.TB, dir string, create func(badger.Options) (*badger.DB,
 	opts := badger.
 		DefaultOptions(dir).
 		WithKeepL0InMemory(true).
-		WithLogger(nil)
+		WithLogger(nil).
+		WithCompression(options.Snappy).
```

Comment:
have to duplicate the code, otherwise there is cycle reference

Reply:
can you add a comment that this is copied.